lib/ex_plasma/output/type/abstract_payment.ex

```diff
@@ -5,14 +5,20 @@ defmodule ExPlasma.Output.Type.AbstractPayment do
 
   @behaviour ExPlasma.Output
 
+  alias __MODULE__.Validator
```

Comment:
why do you use `__MODULE__` instead of full module name? I think `__MODULE__` is meant to be used in the same module.
Validator module will be harder to find by its full module name

Reply:
Not sure I understand you here. It's just the alias.